### PR TITLE
87 logout functionality

### PR DESCRIPTION
**Pull request by hanalaydrus**
#87

### Breakdown:
- [x] Log Out Button
- [x] Clear access_tokens, refresh_tokens, and role_id in asyncStorage

### CODE REVIEW

File: app/components/Button/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default class Button extends Component {
   onClickButton = (inputFunction) => {
     this.setState({ onClick: true })
     inputFunction()
-    setTimeout(() => { this.setState({onClick: false}); }, 2000);
+    // setTimeout(() => { this.setState({onClick: false}); }, 2000);
```
Comment (andy-shi88) [inline app/components/Button/index.js]: clean this up please @hanalaydrus

File: app/containers/Main/index.js [hunk 2]
```diff
@@ -34,9 +42,10 @@ class Main extends Component {
   render() {
     const { fields, isLoggedIn } = this.props;
     const { username, password } = fields || '';
-    if (isLoggedIn) {
-      Actions.mainTabs({ profileData: this.props.profileData })
-    }
+    // if (isLoggedIn) {
```
Comment (andy-shi88) [inline app/containers/Main/index.js]: here too.

File: app/containers/Profile/index.js [hunk 3]
```diff
@@ -42,25 +42,32 @@ class Profile extends Component {
...
   }
 
+  componentWillReceiveProps(prevProps) {
+    if (prevProps.isProfileUpdated !== this.props.isProfileUpdated) {
+      Alert.alert('Success', 'Profile changed');
+      this.props.updateIsProfileUpdated(false)
+    }
+    if (prevProps.isLogOut !== this.props.isLogOut) {
+      Actions.main()
+      console.log('propss', this.props.isLogOut)
```
Comment (andy-shi88) [inline app/containers/Profile/index.js]: hm, can you remove this too? I believe this log is  not important right?

File: config/local.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 export default {
-  API_BASE_URL: 'http://192.168.40.6:5000',
+  API_BASE_URL: 'http://192.168.40.82:5000',
```
Comment (andy-shi88) [inline config/local.js]: try your best not to push `local.js` if it's not needed, everyone should have their own `local.js` saved. :+1:

**Review by andy-shi88 — CHANGES_REQUESTED** [review]
remove any unused comments or log during development process @hanalaydrus

**Comment by hanalaydrus** [comment]
@andy-shi88 Done Mr.Andy

**Review by andy-shi88 — APPROVED** [review]

**Comment by andy-shi88** [comment]
thanks @hanalaydrus

**Pull request merged**